example/api_context_save_example.php

```diff
@@ -15,5 +15,6 @@
 const DEVICE_DESCRIPTION = 'Server 1';
 const PERMITTED_IPS = [];
 
+date_default_timezone_set('UTC'); // When using php5.6
```

Comment:
@dnl-blkv it appears that UTC is returned. But this is not documenten on the docs 🤔.